packages/beacon-node/src/api/impl/beacon/state/utils.ts

```diff
@@ -83,6 +83,51 @@ export async function getStateResponseWithRegen(
   return res;
 }
 
+function mapToGeneralStatus(subStatus: string): string {
```

Comment:
switch-case would be nicer here, we can also reuse `ValidatorStatus` type, and potentially a extended type which includes top-level / general statuses as well